.gitignore

```diff
@@ -48,6 +48,7 @@ doc/reference/toctree/asset/elephant.asset.synchronous*
 doc/reference/toctree/parallel
 doc/reference/toctree/statistics
 doc/reference/toctree/unitary_event_analysis
+doc/reference/toctree/causality
```

Comment:
Can't we just put `doc/reference/toctree` here?

Reply:
I cannot find a reliable regex that would git-ignore everything except toctrees of ASSET and kernel classes. I can revise the problem again, but not in this PR.